hashin.py

```diff
@@ -17,6 +17,19 @@
 else:
     from urllib import urlopen
 
+    if sys.version_info < (2, 7, 9):
+        import warnings
+        warnings.warn(
+            "In Python 2.7.9, the built-in urllib.urlopen() got upgraded "
+            "so that it, by default, does HTTPS certificate verification. "
+            "All prior versions do not. That means you run the risk of "
+            "downloading from a server that claims (man-in-the-middle "
+            "attack) to be https://pypi.python.org but actually is not. "
+            "Consider upgrading your version of Python and/or assert that "
+            "your connection with https://pypi.python.org is valid on "
```

Comment:
I'm not sure how one would go about asserting that. A smart MITM would even check user agents and MITM you only when something dumb (like Python) connected. I suggest just removing that bit.